packages/cubejs-client-react/index.d.ts

```diff
@@ -477,25 +483,47 @@ declare module '@cubejs-client/react' {
    * />
    * ```
    */
-  type MemberUpdater = {
-    add: (member: MemberType) => void;
-    remove: (member: MemberType) => void;
-    update: (member: MemberType, updateWith: MemberType) => void;
+
+  type FilterWithExtraFields = Omit<Filter, 'dimension'> & {
+    dimension: TCubeDimension | TCubeMeasure;
+    operators: { name: string; title: string }[];
+  };
+  type TimeDimensionWithExtraFields = Omit<TimeDimension, 'dimension'> & {
+    dimension: TCubeDimension & { granularities: { name: string; title: string }[] };
+  };
+
+  type MemberUpdater<T> = {
```

Comment:
Please move this type to the bottom of the docs block above, as we use it to generate docs.